search-api.md

```diff
@@ -23,12 +23,6 @@ After receiving a request, the remote server can respond in one of two ways:
   * If a compatible version (`vX.Z` where `Z>=Y`) is supported by the remote server, it should provide a response using this version.
   * If no appropriate version is supported by the remote server, it should respond with `Not Acceptable (406)`, containing a JSON body with a description of the error. All responses, including this one, should contain a `Content-Type` header with the latest API version supported by the server. This will enable the user to re-submit the request using this version of the API.
```

Comment:
Hmm, I'm a bit confused. If the client submits a request using v1.0, which *is* supported, then the server should respond with v1.0 (as per the first bullet point). If it isn't supported, I think the server should respond with an error with the Content-Type of the latest version supported by the server. This gives the requester a version to resubmit with, if they so choose. It's not full-fledged version negotiation, but seemed like a reasonable first-pass that provides 99% of the functionality we'd need.